spec/lib/feed_manager_spec.rb

```diff
@@ -164,6 +164,7 @@
         allow(List).to receive(:where).and_return(list)
         status = Fabricate(:status, text: 'I post a lot', account: bob)
         expect(subject.filter?(:home, status, alice)).to be true
+        expect(subject.filter(:home, status, alice)).to be :skip_home
```

Comment:
> I see changes to existing specs to handle the signature change - should there be new spec for the fixed/change list scenario here?

@mjankowski this spot here (and one below) should catch the new changed scenario, or do you want me to write/add `expect(.filter)` to all of these, to catch all of these scenarios?

Reply:
> @mjankowski this spot here (and one below) should catch the new changed scenario

Ah, I see ... from the high level description I was expecting a new example, but I see its more of a behavior change/addition to some of the existing cases.

> Is this enough, or do you want me to write/add `expect(.filter)` to all of these, to catch all of these scenarios?

I'm not sure ... this might be enough. I'd say that if the other ones would break if it accidentally changed, they should also check on `:filter` like these check on `:skip_home` ... but if that other behavior is fine with just the existing `filter?` true/false checks, prob fine to skip.

Reply:
Alright, what is the proper way to get this PR merged then? Are you able to poke this further internally?

(All the threads need to be resolved, and either I can do that, or someone with permissions from mastodon can do that, but i want an a-ok on those if possible)

Reply:
> Alright, what is the proper way to get this PR merged then?

I don't know, and no one will tell me.

> Are you able to poke this further internally?

I doubt it, but I will absolutely try to.

> (All the threads need to be resolved, and either I can do that, or someone with permissions from mastodon can do that, but i want an a-ok on those if possible)

Be the change you want to see in the world, which in this case means go ahead and resolve some conversations.